plugins/secrets/vault/testing.go

```diff
@@ -0,0 +1,85 @@
+package vault
+
+import (
+	"testing"
+
+	"github.com/hashicorp/vault/api"
+	vaulthttp "github.com/hashicorp/vault/http"
+	vaulttest "github.com/hashicorp/vault/vault"
```

Comment:
I had to call this vaulttest due to `vault` being a type defined in `vault.go`. I'm not sure how this will effect vendoring deps or the godoc for this package. Any thoughts?